torch/testing/_internal/common_optimizers.py

```diff
@@ -1455,6 +1455,18 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
             DecorateInfo(
                 skipIfMps, "TestOptimRenewed", "test_can_load_older_state_dict"
             ),
+            DecorateInfo(
+                toleranceOverride(
+                    {
+                        torch.complex64: tol(
+                            rtol=4.5e-5,
+                            atol=5e-5,
```

Comment:
What is the default tolerance for complex64? Souds like it should be at that level already?

Reply:
rtol of 1.3e-6 and atol of 1e-5

ref: https://pytorch.org/docs/stable/testing.html#torch.testing.assert_close

Reply:
Ho right! Sorry I confused myself with halving the bitlength. This is expected to match fp32.
The general rule here is that for single op, we expect these to hold. If you test a bunch of ops chained one after the other, then we might have to increase the tolerance yes.